spec/models/robot/generator/hamilton_spec.rb

```diff
@@ -7,7 +7,7 @@
 
   let(:expected_output) { expected_file.read }
 
-  before { create(:stock_plate, barcode: 12_345) }
+  # Should update these tests to use SQPD? You can still use DN plates so think this is fine
```

Comment:
Not sure but decide and remove this comment from the code :)

Reply:
Yes, the previously generated barcodes are all valid.